azure-pipelines.yml

```diff
@@ -409,4 +408,59 @@ jobs:
           python ./tool_distributed_mutex.py unlock "$(MUTEX_URL)"
         name: 'Release_the_distributed_mutex'
         # this runs even if the job was canceled (only if the mutex was acquired by this job)
-        condition: and(always(), eq(variables['isMutexSet'], 'true'))
\ No newline at end of file
+        condition: and(always(), eq(variables['isMutexSet'], 'true'))
+
+  - job: Test_Scenarios
```

Comment:
This can be renamed to Additional E2E Tests

Reply:
will update it